lib/models/postgrest_response.dart

```diff
@@ -0,0 +1,57 @@
+// To parse this JSON data, do
+//
+//     final postgrestResponse = postgrestResponseFromJson(jsonString);
```

Comment:
where is this method `postgrestResponseFromJson`? 

Could you use dartdoc syntax `///` to provide a description for this class.

Reply:
I will add it and use dartdoc syntax too